moviepy/video/io/VideoFileClip.py

```diff
@@ -108,7 +111,7 @@ def __init__(
         self.size = self.reader.size
         self.rotation = self.reader.rotation
 
-        self.filename = self.reader.filename
+        self.filename = filename
```

Comment:
is this correct?

Reply:
Yes, it just seemed to make more sense to set `self.filename` directly instead of passing it into the new `self.reader`, and then get it back out again. It's still works identically.